File: PE7/PE7.py
def isprime(n):
    if (n < 2):
        return False
    for i in range(2, int(n**0.5+1)):
        if (n % i == 0):
            return False
    return True

count = 0
i = 1
while (count < 10001):
    i += 1
    if (isprime(i)):
        count += 1
print (i)

File: PE7/PE7.java

public class PE7 {

	public static boolean isprime(int n) {
		if (n < 2) {
			return false;
		}
		boolean is = true;
		for (int i = 2; i <= (int)Math.pow(n, 0.5); i++) {
			if (n % i == 0) is = false;
		}
		return is;
	}
	
	public static void main(String args[]) {
		int count = 0;
		int i = 1;
		while (count < 10001) { // third prime
			i++;
			if (isprime(i)) count++;
		}
		System.out.println(i);
	}
	
}
